pynecone/app.py

```diff
@@ -64,10 +66,23 @@ def __init__(self, *args, **kwargs):
         self.state_manager.setup(state=self.state)
 
         # Set up the API.
-
         self.api = FastAPI()
-        self.add_cors()
-        self.add_default_endpoints()
+
+        # Set up the Socket.IO AsyncServer.
+        self.sio = AsyncServer(async_mode="asgi", cors_allowed_origins="*")
+
+        # Create the socket app. Note 'event' replaces the default 'socket.io' path.
+        socket_app = ASGIApp(self.sio, socketio_path="event")
```

Comment:
Let's use constants.Endpoint.EVENT instead of a string here (and in the EventNameSpace below)

Reply:
I'll do that. Note that `str(constants.Endpoint.EVENT)` returns `"/event"` not `"event"` so I'll also update state.js accordingly, since those have to match.

Reply:
Actually that might cause a bunch of confusion in the future, for example you call ping with `'ping'` but it returns on `'\ping'`. So I'll modify constants.py `Endpoint' to allow it to return the literal value without the leading '\'. 

Now that I think about it it's still confusing. There's also potential confusion between the URL endpoint `/event` and the socket namespace `event`. I'll separate those in constants.py.

Reply:
Now that I think about it it's still confusing. There's also potential confusion between the URL endpoint `/event` and the socket `event` (they're separate concepts in socket.io). I'll separate those in constants.py.

Reply:
Sounds good!